src/operations/mission_controller/mission_controller/node_mission_control.py

```diff
@@ -21,19 +19,8 @@ def __init__(self):
 
         self.create_subscription(State, "/system/as_status", self.callback, 10)
 
-        self.create_service(SelectMission, "select_mission", self.gui_srv)
-
```

Comment:
Just want to make sure that Select Mission wasn't removed only for track day @b1n-ch1kn

Reply:
yeah all good, that was a ros service that wasnt feasible in the state it was in so removed it while i did a little overhaul